lib/active_model_serializers/adapter/json_api/relationship.rb

```diff
@@ -34,6 +34,11 @@ def as_json
 
         private
 
+        def ensure_valid!(json)
```

Comment:
I would say it is not a guess when they are directly telling us this is the case. Are you proposing we wait for them to make this change, then merge this PR? I'm fine if that is the order of operations, though it seems unnecessary imo.

In other words, I want to verify no other problems with this PR other than waiting for them to make the schema update they have told us they are going to make.